docs/sharding-management-sdk.md

```diff
@@ -0,0 +1,53 @@
+# Multiple SignalR service endpoint support in Management SDK
+As a preview feature, we add support for multiple SignalR service instances in Management SDK for **persisent mode only** based on Server SDK, so most content in [Multiple SignalR service endpoint support](sharding.md) is applicable to management SDK. We'll focus on the difference in the following doc.
```

Comment:
add support -> add a support

Reply:
persisent -> Persisent

Reply:
Management SDK, Server SDK -> use full name of the nuget package, and add link

Reply:
most content -> most contents

Reply:
add content table

Reply:
includes the useful content link in the content table. e.g. `Configuration in cross-geo scenarios`

Reply:
add link to https://github.com/Azure/azure-signalr/blob/dev/docs/sharding.md in the table of content

Reply:
I think you can put thic doc into sharing.md, since too much overlap.